docs/tutorials/geodocker-geomesa-spark-on-aws.rst

```diff
@@ -0,0 +1,112 @@
+Geodocker: Bootstrapping GeoMesa Accumulo and Spark on AWS
+==========================================================
+
+Getting started with spatio-temporal analysis with GeoMesa, Accumulo, and Spark on `Amazon Web Services`_ (AWS) is incredibly simple, thanks to the `Geodocker <https://github.com/geodocker/geodocker-accumulo-geomesa>`_ project.  The following guide describes how to bootstrap a GeoMesa Accumulo cluster using `Amazon ElasticMapReduce`_ (EMR) and `Docker`_ and ingesting and querying some sample data.  This guide assumes you have an Amazon Web Services account already provisioned as well as an IAM key pair.  To set up the AWS command line tools, follow the instructions found in the AWS `online documentation <http://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html>`_.
+
+.. _Amazon Web Services: https://aws.amazon.com/
+
+.. _Amazon ElasticMapReduce: https://aws.amazon.com/emr/
+
+.. _Docker: https://www.docker.com/
+
+Bootstrap an EMR cluster
+------------------------
+
+Use the following command to bootstrap an EMR cluster.  You will need to change the ``KeyName`` to the IAM key pair you intend to use for this cluster.  You can also edit the instance types to a size appropriate for your use case.  Make sure you adjust the Accumulo cache configuration settings accordingly.  For instance, if you use a high memory instance type, you will want to increase the ``TSERVER_XMX`` parameter which controls the amount of heap space allocated to the JVM running the Accumulo Tablet Server.  Additionally, you should adjust the ``TSERVER_CACHE_DATA_SIZE`` and the ``TSERVER_CACHE_INDEX_SIZE`` to appropriate fractions of the ``TSERVER_XMX`` memory to take advantage of the increase in memory.
+
+.. code-block:: shell
```

Comment:
In some cases where the code block contains a lot of output (i.e. non-valid Bash code), the Sphinx lexer fails to parse the block and prints an error message on the build console. ``shell`` is slightly more permissive, but probably we should just change it to ``::`` or ``.. code::``.